bitcoin-explained/breaking-down-taproot-activation-options.md

```diff
@@ -0,0 +1,1674 @@
+---
+title: "Breaking Down Taproot Activation Options"
+transcript_by: Sjors via rev.com
+revision_by: edilmedeiros
+media: https://youtu.be/mT0t8Jm0m5E?si=ITfvLBQkhA6KA7DG

```

Comment:
better to use the full link, not sure what this "ITfvLBQkhA6KA7DG" is about
```suggestion
media: https://www.youtube.com/watch?v=mT0t8Jm0m5E
```

Reply:
This is the link Youtube autogenerates with its 'share' option.
But ok, I'll follow this standard.